src/scrollspy/scrollspy.ts

```diff
@@ -0,0 +1,51 @@
+import {Directive, ElementRef, Injectable, Input, OnDestroy, OnInit, Renderer} from '@angular/core';
+
+@Directive({selector: '[ngbScrollSpy]'})
+export class NgbScrollSpy implements OnDestroy, OnInit {
+  active: boolean;
+  selector: string;
+
+  constructor(public elementRef: ElementRef, private scrollSpier: NgbScrollSpier) {}
+
+  @Input()
+  set ngbScrollSpy(selector: string) {
```

Comment:
Why are we using a getter / setter here instead of `@Input('ngbScrollSpy') selector: string;`?